Cargo.toml

```diff
@@ -21,7 +21,7 @@ bio-types = ">=0.9"
 byteorder = "1.3"
 custom_derive = "0.1"
 derive-new = "0.5"
-hts-sys = {version = "2.1.1", default-features = false}
+hts-sys = {version = "2.1.4", default-features = false, features = ["bindgen"]}
```

Comment:
Was this an intentional change that `bindgen` changed from an optional dependency to a required dependency of `rust-htslib`?